packages/fast-components-react-msft/src/subheading/README.md

```diff
@@ -0,0 +1,8 @@
+# Subheading
+Use various subheading sizes to create a clear hierarchy within your page. Subheadings are paired with HTML heading elements such as `<h1>` through `<h6>` and `<p>`.

```

Comment:
Can you update this to be more applicable to the React ecosystem? "Subheadings are generally paired with the heading component."